BostonData/Intents/get_alerts.py

```diff
@@ -0,0 +1,32 @@
+from bs4 import BeautifulSoup
+import urllib.request
```

Comment:
We already include the `requests` library in the zip file, so we might to be consistent with our http calls.